src/graphics/render_wavefront.cpp

```diff
@@ -606,9 +586,8 @@ Convert graphical object into Wavefront object file.
 
 struct Export_to_wavefront_data
 {
-	char *file_basename;
-	char *file_path;
-	FILE *wavefront_file;
+    std::vector<std::string> object_names;
+    std::vector<std::string> objects;
```

Comment:
Consider making this a class with a constructor like for STL, even if only minimally used for now.